gems/pending/spec/util/miq_logger_processor_spec.rb

```diff
@@ -146,4 +180,9 @@
 
     it_should_behave_like "all line processors"
   end
+
+  it "calling read_csv" do
```

Comment:
Don't worry about changing now, but FYI, the general pattern in rspec to describe the method under test is to use `it "#read_csv" do` for instance methods and `it ".read_csv" do` for class methods (the `#` and `.` come from Ruby standard documentation).  That's what I was referring to as well in my other comment.